pkg/validator/validators.go

```diff
@@ -136,9 +160,52 @@ func ingressBackendValidator(req *admissionv1.AdmissionRequest) (*admissionv1.Ad
 		}
 	}
 
+	err := kc.checkDuplicateBackends(ingressBackend)
+	if errors.Is(err, errIngressBackendDuplicateBackends) {
+		return nil, err
+	}
+
+	// Fail open for transient errors, but log
+	if err != nil {
+		log.Error().Err(err).Msg("transient error encountered checking duplicate IngressBackend backends")
```

Comment:
This log message seems ambiguous. Could you reword this to remove the usage of the word `transient`.

Reply:
The error path here should be deterministic. I see why it's not the case now as you are using `List()` below, but we could simply rely on the cache of objects already present in `policy/client.go`.